app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/miband/MiBand2Support.java

```diff
@@ -104,12 +108,25 @@ public void onReceive(Context context, Intent intent) {
 
             } else if (s.equals(BatteryInfoProfile.ACTION_BATTERY_INFO)) {
                 handleBatteryInfo((nodomain.freeyourgadget.gadgetbridge.service.btle.profiles.battery.BatteryInfo) intent.getParcelableExtra(BatteryInfoProfile.EXTRA_BATTERY_INFO));
+            } else if (s.equals(DeviceService.ACTION_MIBAND2_AUTH)) {
+                byte[] response = intent.getExtras().getByteArray(DeviceService.EXTRA_MIBAND2_AUTH_BYTE);
+                BluetoothGattCharacteristic temp = getCharacteristic(MiBand2Service.UUID_UNKNOWN_CHARACTERISTIC9);
+                temp.setValue(response);
+                if ((temp.getProperties() & (BluetoothGattCharacteristic.PROPERTY_WRITE | BluetoothGattCharacteristic.PROPERTY_WRITE_NO_RESPONSE)) != 0) {
+                    Log.d("HELLO", "its looks ok");
```

Comment:
Can you please use the static slf4j LOG variables instead of android.util.Log directly?

We use slf4j to log both to android.util.Log and also to a file that can be inspected later (logging to file is configurable).